gems/pending/disk/modules/VhdxDiskProbe.rb

```diff
@@ -0,0 +1,41 @@
+# encoding: US-ASCII
+
+
+VHDX_DISK      = "VhdxDisk"
+VHDX_SIGNATURE = "vhdxfile"
+
+module VhdxDiskProbe
+  def VhdxDiskProbe.probe(ostruct)
```

Comment:
Any reason why this class method and others below are not specified in the more standard `def self.probe(ostruct)`?